x-pack/filebeat/processors/decode_cef/cef/cef.go

```diff
@@ -13,9 +13,9 @@ import (
 )
 
 // Parser is generated from a ragel state machine using the following command:
-//go:generate ragel -Z -G1 parser.rl -o parser.go
+//go:generate ragel-go -G1 parser.rl -o parser.go
```

Comment:
The diff is much larger than I would have expected for this change. What ragel version did you use?

6.10 from http://www.colm.net/open-source/ragel/ (which is also what [brew gives](https://formulae.brew.sh/formula/ragel))?

Reply:
I'm using the latest version. 7.0.4
![image](https://github.com/user-attachments/assets/a45ace0b-ccaf-4cb0-b712-aabaab708883)

Reply:
FWIW, parser.go and parser_recover.go are generated files. Those two have major changes among all the files.  The only manual change to those two is the comment stating Elastic License and fixed vetting error.

Reply:
If the website is correct then 6.10 is the stable version and 7.0.4 is a development version (unstable). We should use the stable version.

Reply:
Sure, let me downgrade it and regenerate the files

Reply:
Hey @andrewkroh - seems like v6.10 was the key. Now the generated files have only a few changes. 
And `-Z` option is now available in ragel;  `ragel-go` is a v7 thing.

Reply:
I've updated the code. Additionally, should we mention somewhere that we are allowing extra chars on top of CEF Naming guidelines?